certified-connectors/DocuSignDemo/script.csx

```diff
@@ -399,7 +399,21 @@ public class Script : ScriptBase
     body["requiresAcknowledgement"] = "true";
     body["urlToPublishTo"] = notificationProxyUri.AbsoluteUri;
     body["name"] = original["name"]?.ToString();
-    body["envelopeEvents"] = original["envelopeEvents"]?.ToString();
+
+    var envelopeEvent = original["envelopeEvents"]?.ToString();
+    var envelopeEventsArray = new JArray();
+    envelopeEventsArray.Add(envelopeEvent);
+    body["envelopeEvents"] = envelopeEventsArray;
+
+    body["configurationType"] = "custom";
+    body["deliveryMode"] = "sim";
+    body["restv21"] = "true";
+    body["eventData"] = new JObject
+    {
+        ["version"] = "restv2.1",
+        ["format"] = "",
+        ["includeData"] = new JArray()
+    };
```

Comment:
@sriharip-docusign Could you please submit a few screenshots (with environment and flow ids) of the successful runs of this connector update/hotfix? I want to make sure that this hotfix is not breaking any existing flows.